Person.cs

```diff
@@ -0,0 +1,47 @@
+﻿using PhoneBook.StaticHelpers;
+using System.Collections.Generic;
+
+namespace PhoneBook
+{
+    public class Person
+    {
+        private const int rowCountWithSurname = 4;
```

Comment:
instead of `rowCountWithSurname` name it as `membersCount` and `membersCountWithoutSurname`